docs/source/install/index.rst

```diff
@@ -124,11 +124,12 @@ for the same purpose.
       cmake -DUSE_CUDA=ON ..
       make -j4
 
-Finally, install the Python binding.
+Finally, upgrade the python depedencies, then call for Python binding.
```

Comment:
it should be "install" instead of "call for".

Reply:
fixed